# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1349275421 14400
#      Wed Oct 03 10:43:41 2012 -0400
# Node ID cb49817af45730562ae0ebe3ff029ee2c2fc53f4
# Parent  85e3ebc11da4bf187ee6599e8f4e641da68cfb9f
ensure $(octetcdir) exists when installing built-in-docstrings file

* libinterp/Makefile.am (install-built-in-doctstrings): Ensure
$(DESTDIR)$(octetcdir) exists.
From Dirk Ullrich <dirk.ullrich@gmail.com>.

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -310,16 +310,17 @@ uninstall-oct:
 	for f in $(notdir $(OCT_FILES)); do \
 	  rm -f $(DESTDIR)$(octfiledir)/$$f; \
 	done
 	rm -f $(DESTDIR)$(octfiledir)/PKG_ADD
 endif
 .PHONY: install-oct uninstall-oct
 
 install-built-in-docstrings:
+	$(MKDIR_P) $(DESTDIR)$(octetcdir)
 	$(INSTALL_DATA) DOCSTRINGS $(DESTDIR)$(octetcdir)/built-in-docstrings
 .PHONY: install-built-in-docstrings
 
 CLEANFILES = \
   $(DLDFCN_PKG_ADD_FILE) \
   interpfcn/graphics-props.cc \
   parse-tree/oct-parse.output
 
